spring-boot-examples/pom.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.dapr</groupId>
+        <artifactId>dapr-sdk-parent</artifactId>
+        <version>1.14.0-SNAPSHOT</version>
```

Comment:
Update the release script to update any new pom files. This is the disadvantage of creating a new project.

Reply:
@artursouza I've updated only the update sdk script.. do I need to update somewhere else?